nightshift/tasks.py

```diff
@@ -0,0 +1,393 @@
+# -*- coding: utf-8 -*-
+
+"""
+This module provides the manager methods to perform particular tasks
+"""
```

Comment:
I'd suggest turning this all into a class that can be instantiated with a db session, library, libraryId, etc. This makes it a bit easier to interact with since I believe that it is only called from a single file. It simplifies the calls then because for each method here the only parameter that would be passed in would be the array of resources. This should make maintenance easier in the future as well, otherwise this potentially exposes a bunch of methods needing to be changed individually if something basic changes above.

Reply:
I understand. Will work on it, but this seems like a larger task that will take some time for me. I will try to handle it in the next PR if you don't mind.
Created issue https://github.com/BookOps-CAT/NightShift/issues/17